src/pages/en/guides/migration-guide/from-gatsby.md

```diff
@@ -0,0 +1,1199 @@
+---
+title: Migrating from Gatsby
+description: Tips for migrating an existing Gatsby project to Astro
+layout: ~/layouts/CMSLayout.astro
+stub: false
+framework: Gatsby
+---
+
+Here are some tips for converting a Gatsby project to Astro. This is not a full, step-by-step walkthrough, but it will guide you through some changes you will have to make. 
+
+## Key Similarities
+
+While we'll touch on the differences between Gatsby and Astro shortly, it's important to note the ways in which the two overlap.
```

Comment:
The following suggestion was originally written by @sarah11918:

```suggestion
Gatsby and Astro share some similarities that will help you migrate your project:
```

Not opposed to an introductory sentence here, but also not super convinced that we need one. (Good for SEO?) Also would consider `## Key Similarities between Gatsby and Astro` in case we think that is better heading/Algolia/SEO usage.

A little note: I'm less-inclined to narrate the structure of the page (e.g. tell them what's coming up). I think this page is already gonna be quite long, there's a TOC in the sidebar, and people will get from this heading that this section is `similarities` (which is why I also think we might not need any sentence at all? Esp if we put the two names in the heading).

Thoughts?

Reply:
Yeah, totally agree with you here @sarah11918 - the "narrate the structure" wasn't my favorite either. I was really struggling to throw this sentence together, but your edit works nicely.

I did want to keep an introductory sentence here to make the glanceability a bit nicer for the end user.